clusters/l3-sqnc/keycloak/nginx/values.yaml

```diff
@@ -0,0 +1,27 @@
+watchIngressWithoutClass: true
+scope:
+  enabled: true
+ingressClassResource:
+  name: nginx-keycloak
+  default: false
+service:
+  type: LoadBalancer
+  annotations: 
+    service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path: /healthz
```

Comment:
typeo?

Reply:
No, `/healthz` is the location of the `nginx-ingress-controller` health endpoint.